backend/server/routers/model.py

```diff
@@ -37,7 +37,7 @@ class CourseDetails(BaseModel):
     gen_ed: bool
     is_legacy: bool
     is_accurate: bool
-    is_multiterm: bool
+    is_multiterm: Optional[bool]
```

Comment:
`NotRequired` may be better here?

Reply:
NotRequired is actually not allowed in this context